scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -77,7 +85,10 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
     # Calculate the tools and game build paths
     default_base_build_path = ctx.engine_path / 'build' if engine_centric else ctx.project_path / 'build'
     if not tools_build_path:
-        tools_build_path = default_base_build_path / 'tools'
+        if is_installer_sdk:
+            tools_build_path = ctx.engine_path / 'bin' / exp.get_platform_installer_folder_name(selected_platform) / 'profile/Default'
```

Comment:
Don't we have the config stored in a variable?

Otherwise I think we should add a tool_config parameter that we then substitute for "profile" instead of hardcoding it.
```suggestion
            tools_build_path = ctx.engine_path / 'bin' / exp.get_platform_installer_folder_name(selected_platform) / '{tool_config}/Default'
```

Reply:
What was the resolution to this? If we go ahead and do release tools, this line will break without this change